packages/node_modules/@ciscospark/widget-space/src/enhancers/setup.js

```diff
@@ -30,14 +37,29 @@ function setup(props) {
     && sparkState.get('registered')
     && !sparkState.get('hasError')) {
     if (!widgetStatus.isFetchingSpaceDetails && !errors.get('hasError') && !spaceDetails) {
-      if (spaceId) {
-        props.getSpaceDetails(sparkInstance, spaceId, props.intl);
+      // If the selector isn't returning a destination object and we have props for them, store
+      if (destination) {
+        // Use destination object from store to fetch space details
+        // Instead of using props because of legacy prop support
+        props.getSpaceDetails(sparkInstance, destination.id, destination.type, props.intl);
+      }
+      else if (destinationType && destinationId) {
+        props.storeDestination({type: destinationType, id: destinationId});
+      }
+      else if (spaceId) {
+        // eslint-disable-next-line no-console
+        console.warn('The Space ID property is deprecated. Please use Destination ID and Type instead.');
```

Comment:
Since we have access to the sdk's logger here, we should probably use that since it obeys log level flags.